app/routes/__index/dmrv.tsx

```diff
@@ -0,0 +1,34 @@
+import type { LoaderFunction } from "@remix-run/node";
+import { Response } from "@remix-run/node";
+import { json } from "@remix-run/node";
+import { useLoaderData } from "@remix-run/react";
+import Tracking from "~/components/Project/Content/Tracking";
+import type { Dmrv } from "~/types/dmrv";
+
+export const loader: LoaderFunction = async ({
+    params
+  }) => {
+    try {
+      const dmrv = await fetch(`${process.env.DMRV_API}/sea-starktest-kenya-testnet`, {});
```

Comment:
is the slug always the same ?

Reply:
I created it because the indexer was unavailable to test.
I'll delete it.